doc/explanation/performance_tuning.md

```diff
@@ -7,6 +7,22 @@
 
 ## Tune server settings
 
+The vast majority of Linux distributions do not come with optimized
+kernel settings suitable for the operation of a large number of
+containers. The instructions in this document cover the most common
```

Comment:
Using "instances" here rather than "containers" may be more suitable as certainly some settings are only relevant to containers, but many are applicable to both. I like how you've used "container" already in the sub-headings for container specific settings.

Reply:
Changed already, since that was the old text (probably from before VMs ;) ).